src/SelfService/Application/ConfigurationLevelService.cs

```diff
@@ -0,0 +1,116 @@
+using SelfService.Domain.Models;
+using System.Linq;
+
+namespace SelfService.Application;
+
+public enum ConfigurationLevel
+{
+    None,
+    Partial,
+    Complete
+}
+
+public class ConfigurationLevelDetail : Entity<ConfigurationLevelDetail>
+{
+    public ConfigurationLevel level { get; set; }
+    public string description { get; set; }
+    public string suggestion { get; set; }
+    public bool isFocusMetric { get; set; }
+
+    public ConfigurationLevelDetail(ConfigurationLevel level, string description, string suggestion, bool isFocusMetric)
+    {
+        this.level = level;
+        this.description = description;
+        this.suggestion = suggestion;
+        this.isFocusMetric = isFocusMetric;
+    }
+}
+
+public class ConfigurationLevelInfo : Entity<ConfigurationLevelInfo>
+{
+    public ConfigurationLevel overallLevel { get; set; }
+    public bool IsFocusMetric { get; set; }
+    public List<ConfigurationLevelDetail> breakdown { get; set; }
+
+    public ConfigurationLevelInfo()
+    {
+        overallLevel = ConfigurationLevel.None;
+        IsFocusMetric = false;
+        breakdown = new List<ConfigurationLevelDetail>();
+    }
+
+    public void Add(ConfigurationLevelDetail detail)
+    {
+        var existingDetail = breakdown.FirstOrDefault(d => d.description == detail.description);
+        if (existingDetail != null)
+        {
+            breakdown[breakdown.IndexOf(existingDetail)] = detail;
+        }
+        else
+        {
+            breakdown.Add(detail);
+        }
+    }
+}
+
+public class ConfigurationLevelService : IConfigurationLevelService
+{
+    public async Task<ConfigurationLevelInfo> ComputeConfigurationLevel(CapabilityId capabilityId)
+    {
+        var configLevelInfo = new ConfigurationLevelInfo();
+        configLevelInfo.Add(
+            new ConfigurationLevelDetail(
+                await GetKafkaTopicConfigurationLevel(),
+                "kafka-topics-schemas-configured",
```

Comment:
This is fine for now, but I would consider using proper strings to describe this.
Either that, or add some other field for this, as we need to render something human readable in the frontend :)